packages/react-components/react-breadcrumb-preview/stories/BreadcrumbButton/BreadcrumbButtonAppearanceSubtle.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from 'react';
+import {
+  Breadcrumb,
+  BreadcrumbItem,
+  BreadcrumbButton,
+  BreadcrumbDivider,
+  BreadcrumbProps,
+} from '@fluentui/react-breadcrumb-preview';
+import { bundleIcon, CalendarMonth20Filled, CalendarMonth20Regular, GridDots20Regular } from '@fluentui/react-icons';
+import { ButtonProps } from '@fluentui/react-components';
+
+const CalendarMonth = bundleIcon(CalendarMonth20Filled, CalendarMonth20Regular);
+
+type Item = {
+  key: number;
+  item?: string;
+  buttonProps?: {
+    'aria-label'?: string;
+    onClick?: () => void;
+    icon?: ButtonProps['icon'];
+    disabled?: boolean;
+  };
+};
+
+const buttonItems: Item[] = [
+  {
+    key: 0,
+    item: 'Button',
+    buttonProps: {
+      onClick: () => window.open('https://react.fluentui.dev/?path=/docs/components-button-button--default', '_blank'),
+    },
```

Comment:
Is this a way that we're recommending to open a link? In an example below, you're using the `href` property, which seems like the right way.